pajbot/apiwrappers/twitch/helix.py

```diff
@@ -758,3 +758,33 @@ def fetch_all_vips(self, broadcaster_id: str, authorization) -> Set[UserBasics]:
         vips = self._fetch_all_pages(self._fetch_vips_page, broadcaster_id, authorization)
 
         return set(vips)
+
+    def _ban_user(
```

Comment:
The parameters of this function are not easy to understand.

While it strays from some of the norms set up in this class, I think there should be two 'private' functions for using the `Ban User` Helix endpoint:
 - `_ban_user` which omits the `duration` parameter entirely (both in its function definition and in the final API call)
 - `_timeout_user` which requires the `duration` parameter, and forwards it (as expected) to the API call

Reply:
What is the point of having them as internal functions at all? Since they replace the original idea; aren't we better off just making them "public" functions?

If you still suggest that we make them internal functions, do you suggest I also make 2 "public" functions? `ban_user` and `timeout_user`?

Reply:
I'm fine with just making two public functions: `ban_user` and `timeout_user`